src/libs/highlevel/elektra_error.c

```diff
@@ -83,6 +83,8 @@ void elektraErrorAddWarning (ElektraError * error, ElektraError * warning)
  * If no error exists, a pure warning error will be used.
  * @see elektraErrorPureWarning
  *
+ * @note In most cases, errors and warnings were added to the key using src/libs/elektra/errors.c.
```

Comment:
What means "In most cases"?

Reply:
Thanks, I'll update the comment to make it clearer.